lib/internal/structured_clone.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const {
+  MessageChannel,
+  receiveMessageOnPort
+} = require('internal/worker/io');
+
+
+function structuredClone(value, transfer) {
+  // TODO: Improve this with a more efficient solution that avoids instantiating a MessageChannel
+  const channel = new MessageChannel();
```

Comment:
can we instantiate the channel only once (lazily if needed)?

Reply:
yeah I think so!